cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+| <<administration-security-administration-database-privileges, GRANT ... ON DATABASE ... TO>>                                                                                      | Privilege administration      | Assigns a database or schema privilege to one or multiple roles.
+| <<administration-security-subgraph-introduction, GRANT ... ON GRAPH ... [NODES \| RELATIONSHIPS \| ELEMENTS] ... TO>>                                                             | Privilege administration      | Assigns a graph privilege for one or multiple specified elements to one or multiple roles.
+| <<administration-security-administration-dbms-privileges-role-management, GRANT ... ON DBMS TO>>                                                                                  | Privilege administration      | Assigns a DBMS privilege to one or multiple roles.
+| <<administration-security-administration-database-privileges, DENY ... ON DATABASE ... TO>>                                                                                       | Privilege administration      | Denies a database or schema privilege to one or multiple roles.
+| <<administration-security-subgraph-introduction, DENY ... ON GRAPH ... [NODES \| RELATIONSHIPS \| ELEMENTS] ... TO>>                                                              | Privilege administration      | Denies a graph privilege for one or multiple specified elements to one or multiple roles.
+| <<administration-security-administration-dbms-privileges-role-management, DENY ... ON DBMS TO>>                                                                                   | Privilege administration      | Denies a DBMS privilege to one or multiple roles.
+| <<administration-security-roles-revoke, REVOKE ROLE[S] ... FROM>>                                                                                                                 | Privilege administration      | Removes one or multiple roles from one or multiple users.
+| <<administration-security-administration-database-privileges, REVOKE [GRANT \| DENY] ... ON DATABASE ... FROM>>                                                                   | Privilege administration      | Removes a database or schema privilege from one or multiple roles.
+| <<administration-security-subgraph-revoke, REVOKE [GRANT \| DENY] ... ON GRAPH ... [NODES \| RELATIONSHIPS \| ELEMENTS] ... FROM>>                                                | Privilege administration      | Removes a graph privilege for one or multiple specified elements from one or multiple roles
+| <<administration-security-administration-dbms-privileges-role-management, REVOKE [GRANT \| DENY] ... ON DBMS FROM>>                                                               | Privilege administration      | Removes a DBMS privilege from one or multiple roles.
+|===
+
+[[glossary-privileges]]
+=== Privileges
```

Comment:
Should this list also mark which parts of the commands/privileges that are optional?

Reply:
I think it is okay like this. Reads easier. I just did it for the other one because now it looks more similar to the Clauses table :-)